dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgCompanyDataTable.vue

```diff
@@ -53,7 +53,20 @@
             <em class="material-icons" aria-hidden="true" title=""> dataset </em>
           </a>
           <span v-else-if="typeof data[reportingPeriod.dataId] === 'object' && data[reportingPeriod.dataId]?.value">
-            {{ data[reportingPeriod.dataId].value }}
+            <span v-if="isYesNo(data[reportingPeriod.dataId].value) && hasDocument(data[reportingPeriod.dataId])">
+              <DocumentLink
+                :label="yesLabelMap.get(isCertificate(data.kpiLabel))"
+                :download-name="data[reportingPeriod.dataId].dataSource.name"
+                :reference="data[reportingPeriod.dataId].dataSource.reference"
+                show-icon
+              />
+            </span>
+            <span v-else-if="isYesNo(data[reportingPeriod.dataId].value) && isCertificate(data.kpiLabel)">
+              {{ data[reportingPeriod.dataId].value === YesNo.Yes ? "Certified" : "Uncertified" }}
```

Comment:
Why are "Certified" and "Uncertified" hardcoded here for a second time? Can't the YesLabelMap and the NoLabelMap be used again to avoid that the hardcoded string in these Maps and here could diverge?

Reply:
Done